src/views/layouts/main.hbs

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
and here

Reply:
how? 
like this? 
```javascript
{{{> html}}}
```
 ?!

Reply:
you should put all these stuff in head partial

Reply:
remove one curly bracket before the body

Reply:
done.